README.md

```diff
@@ -3,7 +3,7 @@
 Realm Studio is the tool, any developer or system administrator would use when building and maintaining their app built
 on the Realm Mobile Platform.
 
-Visit https://realm.io/products/realm-studio/ for more information on Realm Studio.
+Visit https://docs.mongodb.com/realm/studio/ for more information on Realm Studio.
```

Comment:
I'm I missing something or doesn't the new page contain links to download Studio for the supported platforms?

Reply:
@cbush ?

Reply:
@kraenhansen It does, you just need to hit the "Install" section: https://docs.mongodb.com/realm/studio/install/

Reply:
Ahh .. I was on my phone, where I apparently have to perform an unintuitive horizontal scroll to get to the link.

![Screenshot_20211123-103808](https://user-images.githubusercontent.com/1243959/143001227-4f51e6e8-0379-466c-8c62-a4ec66d9a2ef.jpg)
![Screenshot_20211123-103813](https://user-images.githubusercontent.com/1243959/143001229-2b29c6c7-e8dd-43cc-ab98-cefd0cb63960.jpg)